bullet_train/docs/billing/stripe.md

```diff
@@ -40,98 +85,206 @@ Note this is different than how many Rails engines ask you to install migrations
+STRIPE_PRODUCT_BASIC_ANNUAL_2025_PRICE_ID: price_1QpbQSKKBBBBBBBBBBBBBBBB
+STRIPE_PRODUCT_PRO_ANNUAL_2025_PRICE_ID: price_1QpbQSKKCCCCCCCCCCCCCCCC
+```
+
+### 5. Wire Up Webhooks
 
 Basic subscription creation will work without receiving and processing Stripe's webhooks. However, advanced payment workflows like SCA payments and customer portal cancelations and plan changes require receiving webhooks and processing them.
 
-### 5.1. Ensure HTTP Tunneling is Enabled
+#### 5.1. Ensure HTTP Tunneling is Enabled
 
 Although Stripe provides free tooling for receiving webhooks in your local environment, the officially supported mechanism for doing so in Bullet Train is using [HTTP Tunneling with ngrok](/docs/tunneling.md). This is because we provide support for many types of webhooks across different platforms and packages, so we already need to have ngrok in play.
 
 Ensure you've completed the steps from [HTTP Tunneling with ngrok](/docs/tunneling.md), including updating `BASE_URL` in `config/application.yml` and restarting your Rails server.
 
-### 5.2. Enable Stripe Webhooks
+#### 5.2. Enable Stripe Webhooks
+
+  - Visit [https://dashboard.stripe.com/test/webhooks/create](https://dashboard.stripe.com/test/webhooks/create).
+  - Use the default "add an endpoint" form.
+  - Set "endpoint URL" to `https://YOUR-SUBDOMAIN.ngrok.io/webhooks/incoming/stripe_webhooks`.
+  - Under "select events to listen to" choose "select all events" and click "add events".
+  - Finalize the creation of the endpoint by clicking "add endpoint".
 
- - Visit [https://dashboard.stripe.com/test/webhooks/create](https://dashboard.stripe.com/test/webhooks/create).
- - Use the default "add an endpoint" form.
- - Set "endpoint URL" to `https://YOUR-SUBDOMAIN.ngrok.io/webhooks/incoming/stripe_webhooks`.
- - Under "select events to listen to" choose "select all events" and click "add events".
- - Finalize the creation of the endpoint by clicking "add endpoint".
+*Note:* Depending on your region and when your Stripe account was created, you may be presented with a different admin interface.
+For more details on creating webhooks, please refer to the [Stripe Workbench documentation](https://docs.stripe.com/workbench#webhooks)
 
-### 5.3. Configure Stripe Webhooks Signing Secret
+#### 5.3. Configure Stripe Webhooks Signing Secret
 
 After creating the webhook endpoint, click "reveal" under the heading "signing secret". Copy the `whsec_...` value into your `config/application.yml` like so:
 
 ```yaml
-STRIPE_WEBHOOKS_ENDPOINT_SECRET: whsec_vchvkw3hrLK7SmUiEenExipUcsCgahf9
+STRIPE_WEBHOOKS_ENDPOINT_SECRET: whsec_aBunchOfRandomCharacters
 ```
 
-### 5.4. Test Sample Webhook Delivery
+#### 5.4. Test Sample Webhook Delivery
 
  - Restart your Rails server with `rails restart`.
- - Trigger a test webhook just to ensure it's resulting in an HTTP status code of 201.
+ - Trigger a test webhook using the Stripe CLI. Such as `stripe trigger payment_intent.succeeded`
```

Comment:
We probably want to leave the bit that mentions that we're just looking for a 201 status code. Just to make it clear that we don't expect anything else to happen.